# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1292613472 18000
#      Fri Dec 17 14:17:52 2010 -0500
# Node ID 5e1ec71f73eb82e861ae997450b89aa18252a69f
# Parent  a0278a856516104e709b9aeba1d0ca428732bc3c
ChangeLog entry for previous change

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-12-17  John W. Eaton  <jwe@octave.org>
+
+	* plot/comet.m: Use __plt_get_axis_arg__ to extract axis
+	argument.  Only switch to specified axis temporarily.
+
 2010-12-16  Ben Abbott <bpabbott@mac.com>
 
 	* miscellaneous/unimplemented.m: Remove isprop() form list of
 	unimplemented functions.
 	* plot/isprop.m: New function.
 
 2010-12-16  Olaf Till <olaf.till@uni-jena.de>
 
